variables.tf

```diff
@@ -80,6 +80,12 @@ variable "tags" {
   default     = []
 }
 
+variable "additional_tags" {
```

Comment:
It looks like `var.tags` isn't being used anymore with this change. I think we should remove the old variable to avoid confusion.

We'll also need to be sure to point out this is a breaking change and requires users to update their code.

Reply:
I've removed the old variable.